src/ch02-09-02-erc20-ui.md

```diff
@@ -46,9 +46,10 @@ version = "0.1.0"
 
 [dependencies]
 starknet = ">=2.2.0"
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.8.0-beta.0" }
```

Comment:
Is there any reason to change this file? I mean is is not part of the tutorial is it?